pep-0724.rst

```diff
@@ -0,0 +1,239 @@
+PEP: 724
+Title: Stricter Type Guards
+Author: Rich Chiodo <rchiodo at microsoft.com>, 
+        Eric Traut <erictr at microsoft.com>, 
+        Erik De Bonte <erikd at microsoft.com>
+Sponsor: Jelle Zijlstra <jelle.zijlstra@gmail.com>
+Discussions-To: https://mail.python.org/archives/list/typing-sig@python.org/thread/EMUD2D424OI53DCWQ4H5L6SJD2IXBHUL/
+Status: Draft
+Type: Standards Track
+Topic: Typing
+Content-Type: text/x-rst
+Created: 28-Jul-2023
+Python-Version: 3.13
+Post-History: 
+Resolution: 
```

Comment:
```suggestion
```

Reply:
@rchiodo -- FYI, the lint checks will fail with the 'resolution' header blank -- we expect it to be missing initially, then filled in with a link to a resolution post (once your PEP is accepted, hopefully!).

A